helm/01-check-nginx/assignment.md

```diff
@@ -0,0 +1,65 @@
+---
+slug: check-nginx
+id: o5uzceb0lsbs
+type: challenge
+title: Check that the NGINX chart is installed
+teaser: A setup script has used helm to install an NGINX chart. Verify that it worked!
+notes:
+- type: text
+  contents: Helm is one of the most popular package managers for kubernetes!
+tabs:
+- title: Shell
+  type: terminal
+  hostname: kubernetes-vm
+difficulty: basic
+timelimit: 600
+---
+👀 Verify that Helm is available
+==========================
+
+Helm has been installed on this virtual machine alongside an NGINX ingress controller Helm chart.
+
+You can check that Helm is available and its version with this command:
+
+```bash
+helm version
+```
+
+You will get a message with the build info which should look like this:
+
+```
+version.BuildInfo{Version:"v3.7.2", GitCommit:"663a896f4a815053445eec4153677ddc24a0a361", GitTreeState:"clean", GoVersion:"go1.16.10"}
+```
+
+👀 Verify that the NGINX chart is installed and the pods are running
+==========================
```

Comment:
```suggestion
🚀 Verify that the NGINX chart is installed and the pods are running
====================================================================
```

Let's try and use different Emojis for these to be consistent with other tracks ;) Feel free to find one you like more.

Perhaps we can find a different word than "verify", since it's also the first word in the header above. WDYT?

[Here's a link](https://spec.commonmark.org/0.19/#setext-header-underline ) that describes how setext header underlines work, you should make them the same length as the header. They do work without, but for readability and consistency.